apps/coscrad-frontend/src/components/commands/command-workspace.tsx

```diff
@@ -32,5 +54,6 @@ export const CommandWorkspace = (props: CommandWorkspaceProps): JSX.Element => {
     if (commandResult === Ack)
```

Comment:
We should keep in mind that we may not want to redirect away from the form in case of an error. We might want to keep the form state.

Reply:
I added a comment for now so that it's easy to find this when we are addressing the issue.